src/scc_hypervisor_collector/cli/__init__.py

```diff
@@ -69,6 +73,35 @@ def printlog(log_level: int, error: Exception, logger: logging.Logger) -> None:
             logger.error("ERROR:", exc_info=True)
 
 
+def validate_scc_credentials(scc_credentials_check: bool,
```

Comment:
You could make this a helper method of the SCCUploader class, eliminating the duplication of code for setting up the auth and headers... Such a helper method could end up being the `response = requests.get(...)` line followed by returning true/false based upon whether response.status_code was 200 or not...